Docs/Migration-INI.md

```diff
@@ -938,17 +936,64 @@ Location=12,189
 For completion's sake, below are additional steps required for a complete
 migration (beyond INI changes) to client version [2.11.0.0][client].
 
-### Replace binary files
+### Update client binary files
 
-1. Replace contents of `Resources/Binaries` and `Resources/Compatibility` with
-   new files. This directory contains the .NET Framework 4.8 version
-   of the client.
-2. Copy contents of downloaded `BinariesNET8` into a new directory
+1. Replace `clientdx.exe`, `clientogl.exe` and `clientxna.exe` in `Resources`
+   with new files. Compiled `.pdb` and `.config` files are optional.
+2. Replace contents of `Resources/Binaries` with new files. This directory
+   contains the .NET Framework 4.8 version of the client.
+3. **OPTIONAL** Copy contents of downloaded `BinariesNET8` into a new directory
    `Resources/BinariesNET8`. This directory contains the .NET 8 version
-   of the client.
-3. Replace your launcher executable in game directory (whetever it's called,
-   `YRLauncher.exe`, `MentalOmegaLauncher.exe`, etc.) with version
-   [2.0.7](https://github.com/CnCNet/dta-mg-client-launcher/releases/tag/v2.0.7).
+   of the client that enabled experimental cross-platform Unix support.
+
+The `Resources` directory should look like this (omitting configuration
+files and assets):
+
+```plaintext
+<game dir>/Resources     # override the `Resources` folder to update the client binaries
+├── Binaries             # this folder contains partial .NET 4.8 client files
+├── BinariesNET8         # this folder contains .NET 8.0 client files, where modders can either delete it, or keep it for an experimental cross-platform support
+├── clientdx.exe         # .NET 4.8 client main executable
+├── clientdx.exe.config  # distributed along with `.exe` file. Can be removed but it is better to keep it.
+├── clientdx.pdb         # .pdb file contains debug symbols. It can be either deleted or retained.
+├── clientogl.exe        # .NET 4.8 client main executable
+├── clientogl.exe.config # same as above
+├── clientogl.pdb        # same as above
+├── clientxna.exe        # .NET 4.8 client main executable
+├── clientxna.exe.config # same as above
+└── clientxna.pdb        # same as above
+```
+
+### Update the client launcher
+
+The client launcher (that resides in the game directory) has been updated.
+You can replace the old one with version
+[2.0.7](https://github.com/CnCNet/dta-mg-client-launcher/releases/tag/v2.0.7).
+Remember to rename it from `CncNetLauncherStub.exe` to your launcher name, i.e.
+`YRLauncher.exe`, `MentalOmegaLauncher.exe`. Rename the `.config` file

```

Comment:
> Rename the `.config` file appropriately, i.e. `YRLauncher.exe.config`, `MentalOmegaLauncher.exe.config`.

I think that file was optional, wasn't it?

Reply:
Well, long long ago, the `.exe.config` file for the launcher is not optional, as deleting it causes an additional requirement of .NET 3.5 for Windows 8/10 users, while retaining this file only requires .NET 4.0. DTA preserves this file IIRC.

Some mods like Mental Omega omitted this file so users suffer from installing another .NET runtime. Some users re-distribute the mod with a manually written `.exe.config` file (it basically writes, either try .NET 3.5 first and use .NET 4.0 as a fallback, or try .NET 4.0 first and fallback to .NET 3.5) to overcome it. If we do not include this file, the `.exe` file may have a conflict with user's `.exe.config` file.